src/elements/content-sidebar/activity-feed/version/Version.scss

```diff
@@ -1,11 +1,13 @@
 @import '../../../common/variables';
+@import '../activity-feed/mixin';
 
 .be {
     .bcs-Version {
+        @include bcs-ActivityFeedItem;
+
         display: flex;
         align-items: center;
         justify-content: space-between;
-        width: 100%;
```

Comment:
Is this removal intentional?

Reply:
Yeah there's some issues with the version item styling that I'll address in a separate PR